damus/Views/Zaps/CustomizeZapView.swift

```diff
@@ -166,76 +306,52 @@ struct CustomizeZapView: View {
-                    // User can type in their custom amount, which hides the placeholder.
-                    TextField(selected_amount.amount.formatted(), text: $custom_amount)
-                        .keyboardType(.numberPad)
-                        .onReceive(Just(custom_amount)) { newValue in
-                            if let parsed = handle_string_amount(new_value: newValue) {
-                                self.custom_amount = parsed.formatted()
-                                self.custom_amount_sats = parsed
-                            } else {
-                                self.custom_amount = ""
-                                self.custom_amount_sats = nil
-                            }
-                        }
-                }, header: {
-                    Text("Custom Zap Amount", comment: "Header text to indicate that the text field below it is to enter a custom zap amount.")
-                })
-                
-                Section(content: {
-                    TextField(NSLocalizedString("Awesome post!", comment: "Placeholder text for a comment to send as part of a zap to the user."), text: $comment)
-                }, header: {
-                    Text("Comment", comment: "Header text to indicate that the text field below it is a comment that will be used to send as part of a zap to the user.")
-                })
+            .background(fillColor().edgesIgnoringSafeArea(.all))
+            .onTapGesture {
+                hideKeyboard()
             }
-            .dismissKeyboardOnTap()
-                
-            Section(content: {
-                ZapTypePicker
-            }, header: {
-                Text("Zap Type", comment: "Header text to indicate that the picker below it is to choose the type of zap to send.")
-            }, footer: {
-                Text(zap_type_desc)
-            })
+    }
+
+    var CustomZap: some View {
+        VStack(alignment: .center, spacing: 20) {
             
+            ZapTypeButton()
+                .padding(.top, 50)
             
-            if zapping {
-                Text("Zapping...", comment: "Text to indicate that the app is in the process of sending a zap.")
-            } else {
-                Button(NSLocalizedString("Zap", comment: "Button to send a zap.")) {
-                    let amount = custom_amount_sats ?? selected_amount.amount
-                    send_zap(damus_state: state, event: event, lnurl: lnurl, is_custom: true, comment: comment, amount_sats: amount, zap_type: zap_type)
-                    self.zapping = true
-                }
-                .zIndex(16)
-            }
+            Spacer()
+
+            CustomZapTextField
             
-            if let error {
-                Text(error)
-                    .foregroundColor(.red)
+            AmountPicker
+            
+            ZapReply
+            
+            ZapButton
+            
+            Spacer()
+            
+            Spacer()
```

Comment:
extra spacer here?

Reply:
Pushed it up slightly, but not a huge difference so I removed it